locales/help_enu/gkhCAIndex.html

```diff
@@ -1,25 +1,33 @@
-</ul>
-<p>While you are researching and viewing some information, authenticity level shows to you how much you can trust that information. It is better to set &quot;Questionable reliability of evidence&quot; if a source document had blots, was damaged and could not be read with confidence. This can prevent you from false conclusions.</p>
-<p>If a reference to a source has no authenticity level, such reference gets &quot;Unreliable evidence or estimated data&quot; by default. Just because mentioning a person in document is the important acknowledgement by itself.</p>
-<p>Authenticity index is evaluated automatically for each person basing on authenticity level (as the weighted arithmetic mean). You can view authenticity index
-in <a href="gkhStats.html">statistics</a> module for analysis of mean values or in <a href="gkhTreeChart.html">tree diagrams</a> for pictorial presentation. The index is evaluated using all references to sources, specified
-in both person entity and all materials and events.</p>
+    <h1>Authenticity Index</h1>
+    <p>The authenticity index ("certainty") of information about a person is a special indicator that can be evaluated for person entity only.</p>
+    <p>Each reference to a source in a person entity has a specific authenticity level when mentioning that person in the source. The authenticity level can have the following values:</p>
+    <ul>
+        <li>&quot;Unreliable evidence&quot; indicates a 25% authenticity level.</li>
+        <li>&quot;Questionable reliability of evidence&quot; indicates a 50% level of authenticity.</li>
+        <li>&quot;Circumstantial evidence&quot; is data officially recorded sometime after event (authenticity is 75%).</li>
+        <li>&quot;Direct and primary evidence&quot; is considered a dominance of evidences (authenticity is 100%).</li>
+    </ul>
+    <p>
+        While researching and viewing some information, the authenticity level shows how much the information can be trusted. It is better to set &quot;Questionable reliability of evidence&quot;
+        if a source document had blots or was damaged and could not be read with confidence. This can help prevent false conclusions.
+    </p>
+    <p>
+        If a reference to a source has no authenticity level, the reference is considered &quot;Unreliable evidence or estimated data&quot; by default.
+        The mentioning of a person in a document is an important acknowledgement by itself.
+    </p>
+    <p>
+        The authenticity index is evaluated automatically for each person based on authenticity level (as a weighted arithmetic mean). You can view the authenticity index
+        in <a href="gkhStats.html">statistics</a> module for analysis of mean values or in <a href="gkhTreeChart.html">tree diagrams</a> for pictorial presentation.
+        The index is evaluated using all references to sources&#151;specified in both the person entity and all materials and the associated events.
+    </p>
```

Comment:
The file is changed almost completely, no specific change is visible - a solid "flare". It will not work, you need to correct the formatting back.